app/templates/static/layout.static.pug

```diff
@@ -3,7 +3,7 @@ block begin
 html(lang='en')
   head
     meta(charset='utf-8')
-    meta(name='viewport', content='width=1024')
+    meta(name='viewport', content='width=1024', data-vue-meta="true", data-vmid="viewport")
```

Comment:
This is so cool! So to clarify my understanding, vue will recursively push these properties up into the header?

Reply:
Yeah those two properties allow vue-meta to find this and merge it with the meta tags that it adds so we don't have duplicates.  Turns out that the browser is good at adjusting the viewport when the meta tag changes, it just can't do it with multiple meta tags :).